apps/src/templates/studioHomepages/TeacherHomepage.jsx

```diff
@@ -176,7 +177,8 @@ export class UnconnectedTeacherHomepage extends Component {
       teacherEmail,
       canViewAdvancedTools,
       isEnglish,
-      specialAnnouncement
+      specialAnnouncement,
+      teacherApplications2021
```

Comment:
what's the relationship between `specialAnnouncement` and `teacherApplications2021`? can you not just set `specialAnnouncement` to `false` when `teacherApplications2021` would be false and avoid adding this additional very specific property?

Reply:
`specialAnnouncement` is a hash with the data for the banner which is determined by `announcements.json`, it is passed into the component
```
<SpecialAnnouncementActionBlock
     announcement={specialAnnouncement}
/>
```
If there is no announcement specified for the page it will be nil

Reply:
ahhh, sorry i didn't look closely enough and thought it was a bool. thanks!

Reply:
You're right though, I could set `specialAnnouncement` to nil if the flag isn't on